Reemit.Decompiler.Clr/Metadata/Tables/IMetadataRecord.cs

```diff
@@ -0,0 +1,21 @@
+namespace Reemit.Decompiler.Clr.Metadata.Tables;
+
+public abstract class MetadataRecord : IMetadataRecord
```

Comment:
You might want to move this to `MetadataRecord.cs`.

Reply:
Good idea.